pyodrx/opendrive.py

```diff
@@ -109,7 +109,7 @@ class Road():
                 write a open scenario xml
                 
     """
-    def __init__(self,road_id,planview,lanes, road_type = -1,name=None, rule=None):
+    def __init__(self,road_id,planview,lanes, road_type = -1,name=None, rule=None, signals=None):
```

Comment:
What was the reason to integrate this here? I would have expected an add_signal method similar to the roadmarks for lanes. Is there a preferred way for integration?

Reply:
I usually think that if you can add multiple entries of one class you do a add_ functionality, like roadmarks, or lane_sections to lanes. 
Here you either add signals (which is a single entry) or you add Signal directly to the road (then create the signals in road).
So both "work" in this case, following how the rest is built up.